sdk/python/kfp/dsl/_pipeline.py

```diff
@@ -104,6 +106,21 @@ def set_ttl_seconds_after_finished(self, seconds: int):
     self.ttl_seconds_after_finished = seconds
     return self
 
+  def set_pod_disruption_budget(self, min_available: str, selector: str=None):
+    """ PodDisruptionBudget holds the number of concurrent disruptions that you allow for pipeline Pods.
+	      Controller will automatically add the selector with workflow name, if selector is empty.
```

Comment:
I think this line is more suitable in the args section, to describe the behavior of default selector. WDYT?

Reply:
Yes, agree the sentence ```Controller will automatically add the selector with workflow name, if selector is empty.``` is better suited in the args section. Will move it.